debian/rules

```diff
@@ -26,6 +26,8 @@ else
 enable_tf=false
 endif
 
+enable_py=true
```

Comment:
this is unnecessary, just set meson option -Denable-python=true

Reply:
Got it.